data/bactrian.txt

```diff
@@ -138,7 +138,7 @@ mission "Bactrian Unlock [0]: Haven"
 		has "Bactrian Unlock [0]: active"
 	on offer
 		conversation
-			`You spend a few hours scanning the surface of the planet in an attempt to find the Star Queen, but nothing comes up. It will likely require military-grade scanners in order to find the ship, which Lieutenant Paris will undoubtedly have.`
+			`You spend a few hours scanning the surface of the planet in an attempt to find the Star Queen, but nothing comes up. It will likely require military-grade scanners in order to find the ship, which the Deep Security will undoubtedly have.`
```

Comment:
Anyone have BA/MFA in English here and want to settle this? The answer has to do with definite articles, mass/uncountable nouns, and using mass nouns as proper nouns.

Perhaps we strip it the second sentence of line 141 and find a less-contestable phrasing:
`Perhaps these scans will require the superior technology found aboard Deep Security ships.`